int-ball2_control/scripts/direct_effort_control.py

```diff
@@ -22,38 +22,39 @@ def __init__(self):
         )
 
         # Publisher for force/torque
-        self.wrench_publisher = self.create_publisher(Wrench, "/ctl/wrench", 10)
+        self.wrench_publisher = self.create_publisher(WrenchStamped, "/ctl/wrench", 10)
 
         self.get_logger().info("Int-Ball2 Force Controller Node Initialized")
 
     def joy_callback(self, msg):
         """
         Callback for handling joystick inputs and mapping them to Wrench messages.
         """
-        wrench = Wrench()
+        wrench = WrenchStamped()
+        wrench.header.stamp = self.get_clock().now().to_msg()
 
         # Left stick controls force in X and Y
-        wrench.force.x = msg.axes[1] * self.force_scale    # Left stick X
-        wrench.force.y = msg.axes[0] * self.force_scale    # Left stick Y
+        wrench.wrench.force.x = msg.axes[1] * self.force_scale    # Left stick X
+        wrench.wrench.force.y = msg.axes[0] * self.force_scale    # Left stick Y
 
         # Right stick controls rotation (torque) in X and Y
-        wrench.torque.x = msg.axes[4] * self.torque_scale  # Right stick X
-        wrench.torque.y = msg.axes[3] * self.torque_scale  # Right stick Y
+        wrench.wrench.torque.x = msg.axes[4] * self.torque_scale  # Right stick X
+        wrench.wrench.torque.y = msg.axes[3] * self.torque_scale  # Right stick Y
 
         # L2 (Axis 2) and R2 (Axis 5) control force in Z
         L2 = (1 - msg.axes[2]) / 2  # Convert from [-1, 1] to [0, 1]
         R2 = (1 - msg.axes[5]) / 2  # Convert from [-1, 1] to [0, 1]
         
         if msg.buttons[1]:
-            wrench.force.z  = (R2 - L2) * self.force_scale  # R2 increases, L2 decreases
+            wrench.wrench.force.z  = (R2 - L2) * self.force_scale  # R2 increases, L2 decreases
         elif msg.buttons[0]:
-            wrench.torque.z = (R2 - L2) * self.torque_scale # R2 increases, L2 decreases
+            wrench.wrench.torque.z = (R2 - L2) * self.torque_scale # R2 increases, L2 decreases
 
         # Publish the wrench message
         self.wrench_publisher.publish(wrench)
         self.get_logger().info(
```

Comment:
I have just noticed but we should output the log when debugging only. 
```suggestion
        self.get_logger().debug(
```